<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSelfTrain</class>
 <widget class="QWidget" name="FormSelfTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelfTrain</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="labelFolder">
     <property name="text">
      <string>folder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabDataGet">
      <attribute name="title">
       <string>Data Get</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1,0,0" columnstretch="0">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFile">
         <property name="text">
          <string>file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="tBtnPrev">
           <property name="text">
            <string>prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIndex">
           <property name="text">
            <string>index</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnNext">
           <property name="text">
            <string>next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="tBtnRecord">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>record</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widgetPlot" native="true">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gLayPlot"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDataUpload">
      <attribute name="title">
       <string>Data Upload</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="tableViewFile"/>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelMethod">
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxMethod"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpload">
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTrain">
      <attribute name="title">
       <string>Train</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowserTrainInfo"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
